### PR TITLE
Fix double write of workflow states.

### PR DESCRIPTION
**Pull request by mcovarr**
Removing a duplicate write of workflow state that seemed to be responsible for some Tyburn failures.  The WorkflowActor already handles the persistence of its own state, WorkflowManagerActor doesn't need to be doing this as well.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -46,7 +46,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.slick" %% "slick" % slickV,
   "com.typesafe.slick" %% "slick-hikaricp" % slickV,
   "org.hsqldb" % "hsqldb" % "2.3.2",
-  "com.google.gcloud" % "gcloud-java" % "latest.integration",
+  "com.google.gcloud" % "gcloud-java" % "0.0.9",
```
Comment (mcovarr) [inline build.sbt]: unrelated change

**Comment by scottfrazer** [comment]
removing code while fixing a bug?  :+1: :+1:

**Comment by scottfrazer** [comment]
Second reviewer @kshakir

**Comment by geoffjentry** [comment]
@scottfrazer does that count for two thumbs? :)

**Comment by scottfrazer** [comment]
haha

**Comment by cjllanwarne** [comment]
:+1: :+1: :+1:

**Comment by mcovarr** [comment]
5 thumbs!  Merging posthaste, the public demands it.
